api/build/skip.go

```diff
@@ -9,7 +9,7 @@ import (
 // SkipEmptyBuild checks if the build should be skipped due to it
 // not containing any steps besides init or clone.
 //
-//nolint:goconst // ignore init and clone constants
+
```

Comment:
minor: linter with fix _does some work_ but for stuff like this requires some manual intervention unfortunately. i'd remove line 11 + 12.

Reply:
oof, that's too bad